File: BusinessEnvironmentSnapshots/BusinessEnvironmentSnapshots.Database/Model/Tab.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BusinessEnvironmentSnapshots.Database.Model
{
    public class Tab
    {
        public string Code { get; set; }
        public string Name { get; set; }
        public List<Section> Sections { get; set; }
    }
}

File: BusinessEnvironmentSnapshots/BusinessEnvironmentSnapshots.UI/scripts/src/data/tab.ts
import dsc = require('data/section');

export interface Tab {
    code: string;
    name: string;
    sections: dsc.Section[];
}

export function tabFrom(
    code: string,
    name: string,
    sections: dsc.Section[]
) : Tab {
    return {
        code: code,
        name: name,
        sections: sections
    };
}